uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/leave-request-api.js

```diff
@@ -17,6 +17,27 @@ define([
       return dataFromServer && !!dataFromServer.is_error;
     }
 
+    /**
+     * Checks given leave type and returns appropriate entity name
+     *
+     * @param {String} leaveRequestType whether its leave, sick or toil
+     * @return {String}
+     */
+    function getEntityName(leaveRequestType) {
+      var entityName = 'LeaveRequest';
+
+      leaveRequestType = !!leaveRequestType ? leaveRequestType : 'leave';
+
+      if (leaveRequestType === 'sick') {
+        entityName = 'SicknessRequest';
+      }
+      else if (leaveRequestType === 'leave') {
```

Comment:
this `else` is unnecessary, you already initialized `entityName` as "LeaveRequest"